runtime/testruntime/testruntime.go

```diff
@@ -224,11 +230,20 @@ func NewInstanceForProject(t TestingT, name string) (*runtime.Runtime, string) {
 	return rt, inst.ID
 }
 
-func NewInstanceForDruidProject(t TestingT) (*runtime.Runtime, string) {
+func NewInstanceForDruidProject(t *testing.T) (*runtime.Runtime, string, error) {
+	_, err := os.Stat(".env")
+	if !errors.Is(err, fs.ErrNotExist) {
+		require.NoError(t, godotenv.Load())
```

Comment:
Related to the comment above – you may need to pass the repo root path here for this to work